src/CloudFlareZoneInterface.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * @file
+ * Contains \Drupal\cloudflare\CloudflareZoneInterface.
+ */
+
+namespace Drupal\cloudflare;
+use CloudFlarePhpSdk\ApiTypes\Zone\ZoneSettings;
+/**
+ * Zone methods for CloudFlare.
+ */
+interface CloudflareZoneInterface {
+
+  /**
+   * Updates the current zone's settings from CloudFlare's API.
+   *
+   * @param \CloudFlarePhpSdk\ApiTypes\Zone\ZoneSettings $zone_settings
+   *   The updated settings object.
+   */
+  public function updateZone(ZoneSettings $zone_settings);
```

Comment:
Below, there's `getZoneSettings()`. Here it says `updateZone()`, but it receives `ZoneSettings`.

Highly inconsistent.